### DISCUSSION TITLE
Welcome to the Brain Observatory Toolbox Discussions!

### DISCUSSION THREAD

**Discussion by emeyers** [emeyers/Brain-Observatory-Toolbox#118] [Announcements]
## 👋 Welcome!

The **Brain Observatory Toolbox** is currently an early-stage project allowing users to easily access and utilize the public Allen Brain Observatory datasets [1]. 

The early contributors look forward to hearing from and supporting early users directly in this Discussion forum. This way all users (and contributors) may benefit from the conversation(s) that ensue.

**Have a question?** Check out the [**Q&A**](https://github.com/emeyers/Brain-Observatory-Toolbox/discussions/categories/q-a) to find answers and ask new questions.

**Have ideas or other thoughts?** Please share below!

<sup>[1] Copyright 2016 Allen Institute for Brain Science. Allen Brain Observatory. Available from: portal.brain-map.org/explore/circuits.</sup>

**Comment by ThalionA** [comment]
Hi, 

Could you please provide a tutorial for accessing the optotagging experiments (something along the lines of https://allensdk.readthedocs.io/en/latest/_static/examples/nb/ecephys_optotagging.html)? Alternatively, is there already a list with putative interneuron IDs for each session that can be accessed directly?

Many thanks,
Theo
